apps/system/utils/utils_cpuload.c

```diff
@@ -109,6 +109,16 @@ static void cpuload_stop(void)
 	}
 }
 
+static bool has_cpuload(char *load_info)
+{
+	int len = strlen(load_info);
```

Comment:
Pls add  a comment here about why you are comparing with "-0.0". If someone changes the format of cpuload, then this will get affected.